test/segment-loader.test.js

```diff
@@ -423,7 +424,7 @@ QUnit.module('SegmentLoader', function(hooks) {
           this.currentTime = 210;
         });
 
-        standardXHRResponse(this.requests.shift(), muxedSegment());
+        standardXHRResponse(this.requests.shift(), oneSecondSegment());
```

Comment:
The probed values for `muxedSegment` are not legitimate it seems. When using  `oneSecond.ts`, a one second ts file that I generated everything probes as expected in the test.